frontend/src/app/modules/home/issues/issue-details/issue-details-page/issue-details-page.component.html

```diff
@@ -52,7 +50,33 @@ <h2 appTooltipFullName>{{issueMessage?.issueDetails.className}}</h2>
                     <app-issue-events [issueMessage]="issueMessage"></app-issue-events>
                 </ng-template>
             </p-tabPanel>
+            <p-tabPanel [disabled]="!solutionItems?.length" header="Stackoverflow solutions">
+                <ng-template pTemplate="content">
+                    <div class="solution-item">
+                        <p-table [value]="solutionItems">
+                            <ng-template pTemplate="header">
+                                <tr>
+                                    <th class="link-field">Link</th>
+                                    <th>Answer count</th>
+                                    <th>Question rating</th>
+                                    <th>View count</th>
+                                    <th>Status</th>
+                                </tr>
+                            </ng-template>
+                            <ng-template let-solutionItem pTemplate="body">
+                                <tr>
+                                    <td><a appTooltipFullName href="{{solutionItem.link}}" target="_blank">{{solutionItem.link}}</a></td>
+                                    <td class="centered">{{solutionItem.answer_count}}</td>
+                                    <td class="centered">{{solutionItem.score}}</td>
+                                    <td class="centered">{{solutionItem.view_count}}</td>
+                                    <td>{{solutionItem.is_answered ? 'Is answered' : 'No answered'}}</td>
```

Comment:
It's better to change prop name to our style. 
https://google.github.io/styleguide/tsguide.html#identifiers
And also, what is '_blank' ??

Reply:
It is a problem to change prop name, because we get data from exchange api in this format.  Target _blank opens link in a new window